client-ui/src/app/components/lineage-overview/lineage-overview-graph/lineage-overview-graph.component.ts

```diff
@@ -70,23 +70,27 @@ export class LineageOverviewGraphComponent implements OnInit, AfterViewInit, OnD
           return this.store
             .select('lineageOverview')
             .pipe(
-              filter(state => !_.isNil(state)),
-              map(state => {
-                return { graph: state, layout: res.layout, contextMenu: res.contextMenu }
-              })
+                filter(state => !_.isNil(state)),
+                map(state => {
+                  return { graph: state, layout: res.layout, contextMenu: res.contextMenu }
+                }),
+                // take(1)
```

Comment:
Do we have some code style for the frontend? In this file the indentation seems to be 2 spaces, it would be nice to keep this consistent.

Reply:
Sorry, did not menthioned that. I have 4 spaces by default. And forgot to switch of to the current project settings.

Reply:
Actually I am getting confused, and my IDE too as one some files having 4 spaces idents, and other 2 ...